src/libcore/option.rs

```diff
@@ -605,6 +605,8 @@ impl<T> Option<T> {
     /// Returns `None` if the option is `None`, otherwise calls `f` with the
     /// wrapped value and returns the result.
     ///
+    /// Other languages call this operation "flatmap".
```

Comment:
Oh, I guess that's US style. I'd rather remove the quotes in that case. Are you ok with that?